exercise-15/src/com/nayema/Actor.java

```diff
@@ -0,0 +1,15 @@
+package com.nayema;
+
+public class Actor {
+    private static String[] name = {"Arnold Schwarzenegger", "Will Smith", "Anglenia Jolie", "Brad Pitt"};
+
+    private Actor() {
+    }
+
+    public static void randomActorName() {
+        int length = name.length;
+        int random = (int)(Math.random() * length);
+        String actorName = name[random];
+        System.out.println(" I am " + actorName + "!");
```

Comment:
This method is doing too many things. Presentation concerns like this should be in `Main` for now as per the Issue instructions.